inst/WORDLIST

```diff
@@ -9,6 +9,7 @@ DM
 DS
 EG
 GlaxoSmithKline
+GRPID
```

Comment:
Do we need this?

Reply:
Yes its needed, have added it back in.

Reply:
Strange, it seems that the backticks are not processed correctly when the man pages are generated. However, most likely nobody will read the man page. Thus we can ignore it.